### ISSUE TITLE
[Minor] overview page: remove "elements"

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-media-guide#18]
The word "element" keeps distracting me. I suggest removing it. If needed, maybe use an alternative like "provision" or "aspect" or such.

**Comment by bakkenb** [comment]
+1... I like "provision."

**Comment by shawna-slh** [comment]
Previous heading was:
> **Elements of Accessible Audio and Video**

I tried just taking out "Elements of"… (and left it like that for a couple hours)
... but that didn't work – I think it needs a noun there.

"Provision of Accessible Audio and Video" didn't work for me at all.
(although when I looked up the definition, it seemed fitting. :-/)

Brainstorms:
•	provision
•	aspects
•	components
•	features
•	pieces
•	parts
•	traits
•	essentials
•	foundations
•	basics

"**Components of Accessible Audio and Video**" seems good. I wonder if any possible confusion given [other components resource](https://www.w3.org/WAI/fundamentals/components/)? (probably not)

I do like "**Aspects of Accessible Audio and Video**". (and nice alliteration. :-)
. I wonder about non-native understand-ability (although Shadi mentioned it:-) and translate-ability.
. "aspects" seems a little wimpier than "components".
I'll try that for now and see how it flies.

Feel free to add other brainstorms, reactions, or input -- and re-open if you'd like.
